File: Revisão Pratica/Comandos de Decisão/relacionais.cpp
#include <iostream>

int main() {
    int idade;
    
    std::cout << "Digite sua idade: ";
    std::cin >> idade;

    if (idade >= 18) {
        std::cout << "Voce e maior de idade." << std::endl;
    } else if (idade >= 12) {
        std::cout << "Voce e um adolescente." << std::endl;
    } else {
        std::cout << "Voce e uma crianca." << std::endl;
    }

    return 0;
}

File: Revisão Pratica/Comandos de Decisão/relacionais.java
import java.util.Scanner;

public class java_decisao {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        System.out.print("Digite sua idade: ");
        int idade = scanner.nextInt();

        if (idade >= 18) {
            System.out.println("Voce e maior de idade.");
        } else if (idade >= 12) {
            System.out.println("Voce e um adolescente.");
        } else {
            System.out.println("Voce e uma crianca.");
        }
        
        scanner.close();
    }
}